src/hooks/useCustomControl.test.tsx

```diff
@@ -0,0 +1,98 @@
+import { ControlProps } from '@jsonforms/core';
+import { renderHook } from '@testing-library/react-hooks';
+import { act } from 'react-dom/test-utils';
+import useCustomControl from './useCustomControl';
+
+describe('useCustomControl()', () => {
+	const handleChangeMock = jest.fn();
+	const props: ControlProps = {
+		data: 'value',
+		errors: 'error',
+		label: 'label',
+		handleChange: handleChangeMock,
+		path: './',
+		id: 'd',
+		rootSchema: {},
+		uischema: { scope: 'test', type: 'Control' },
+		schema: {},
+		enabled: true,
+		visible: true,
+	};
+	test('should return correct entries', () => {
+		const { result } = renderHook(() =>
+			useCustomControl({
+				data: 'test',
+				errors: 'test-error',
+				label: 'test-label',
+				path: '/test',
+			} as ControlProps),
+		);
+
+		const expectedResult = {
+			isFocused: false,
+			isDirty: true,
+			isValid: false,
+			onFocus: function () {
+				console.log('onFocus');
+			},
+			onBlur: () => console.log('onBlur'),
+			onChange: () => console.log('onChange'),
+		};
+		const currResult = result.current;
+
+		expect(Object.keys(currResult)).toEqual(Object.keys(expectedResult));
```

Comment:
This line seems redundant to me. Aren't you explicitly asserting each key has the expected value below?

Reply:
true